integration/flink/README.md

```diff
@@ -0,0 +1,25 @@
+# OpenLineage Flink Agent
+
+The OpenLineage Flink Agent uses jvm instrumentation to emit OpenLineage metadata.
+
+## Installation
+
+Maven:
+
+```xml
+<dependency>
+    <groupId>io.openlineage</groupId>
+    <artifactId>openlineage-flink</artifactId>
+    <version>0.1.0</version>

```

Comment:
You'll want to align the version of the `openlineage-flink` lib with the version of the other integrations.